src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj

```diff
@@ -19,4 +20,9 @@
       <FilesToPackage Include="@(Analyzer)" ExcludeFromValidation="true" TargetPath="analyzers/%(RecursiveDir)" />
     </ItemGroup>
   </Target>
+
+  <ItemGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
+    <ReferencePlatformManifestPath  Include="PlatformManifest.txt" TargetPath="data" GeneratedBuildFile="true" />
```

Comment:
I'm not sure if we need `GeneratedBuildFile` here or not, @jkoritzinsky?

Reply:
Looks like this metadata is required to avoid it showing up in the framework deps file here: https://github.com/dotnet/arcade/blob/0602cfaab1dfdbf7802dd3d2876fd17cf4ae04a7/src/Microsoft.DotNet.SharedFramework.Sdk/src/GenerateSharedFrameworkDepsFile.cs#L57

I think we can avoid the extra item though.